src/pages/resultsView/oncoprint/AddClinicalTracks.tsx

```diff
@@ -150,9 +156,14 @@ export default class AddClinicalTracks extends React.Component<
                 x => x.displayName.toLowerCase()
             );
 
+            sortedAttributes.customCharts = _.sortBy<ExtendedClinicalAttribute>(
+                sortedAttributes.customCharts,
+                x => x.displayName.toLowerCase()
+            );
+
             return Promise.resolve(
                 _.mapValues(sortedAttributes, attrs => {
-                    return attrs.map(attr => ({
+                    return (attrs as ClinicalAttribute[]).map(attr => ({
```

Comment:
Do we need to cast this? I thought `attrs` will derived the type from `sortedAttributes`.

Reply:
its this typescript issue where when the array type is ambiguous then it cant resolve the array methods type (here its either ClinicalAttribute[] or ExtendedClinicalAttribute[], but either one works for the return type)